packages/react-native/ReactAndroid/src/main/jni/react/newarchdefaults/DefaultComponentsRegistry.cpp

```diff
@@ -24,6 +25,10 @@ std::shared_ptr<const ComponentDescriptorProviderRegistry>
 DefaultComponentsRegistry::sharedProviderRegistry() {
   auto providerRegistry = CoreComponentsRegistry::sharedProviderRegistry();
 
+  FBASSERTMSGF(
+      DefaultComponentsRegistry::registerComponentDescriptorsFromEntryPoint,
+      "'registerComponentDescriptorsFromEntryPoint' was not initialized in 'JNI_OnLoad'");
```

Comment:
Can we use instead `#include <react/debug/react_native_assert.h>` and the function `react_native_assert` as we do in other parts of the codebase?